pkg/setting/setting.go

```diff
@@ -1691,6 +1692,16 @@ func readUserSettings(iniFile *ini.File, cfg *Cfg) error {
 		return errors.New("the minimum supported value for the `user_invite_max_lifetime_duration` configuration is 15m (15 minutes)")
 	}
 
+	cfg.UserLastSeenUpdateInterval, err = gtime.ParseDuration(valueAsString(users, "last_seen_update_interval", "15m"))
+	if err != nil {
+		return err
+	}
+
+	if cfg.UserLastSeenUpdateInterval < time.Minute*5 {
```

Comment:
Hey @parambath92 can we add a limit to this update interval to 1h? I'm afraid some auditing features will misbehave if the interval is too large